js/private/js_binary.bzl

```diff
@@ -267,8 +267,7 @@ _ATTRS = {
         attribute based on a `config_setting` rule.
         See /js/private/BUILD.bazel in rules_js for an example.
         """,
-        # TODO(2.0): make this mandatory so that downstream binary rules that inherit these attributes are required to set it
-        mandatory = False,
+        mandatory = True,
     ),
```

Comment:
Can we provide a default or does that always have to be manually specific with the `select` + `@aspect_rules_js//js:enable_runfiles`?

Reply:
The default is not available in the rules context... but this attribute gets deleted further down the stack anyway.